chapters/memory-security/ctf/drills/tasks/hidden-in-plain-sight-2/solution/README.md

```diff
@@ -0,0 +1,30 @@
+# Solution
+
+In a nature similar to that of the previous exercise, we take a close look at the `objdump` disassembly output of the binary using the `objdump -D -M intel link2` command, specifically focusing on the `helper()` function:
```

Comment:
Replace "previous exercise" with `[hidden-in-plain-sight-1](link/to/hidden-in-plain-sight-1)`.